packages/shared/src/tagged-errors.ts

```diff
@@ -53,5 +51,5 @@ export class RetryError extends TaggedError("RetryError") {}
 export class FetchError extends TaggedError("FetchError")<{
   readonly input: RequestInfo | URL;
   readonly error: unknown;
-  readonly data?: Json;
+  readonly data?: any;
```

Comment:
`unknown` is better than `any`, as `any` breaks all type safety, but otherwise perhaps generics are helpful?